src/PockyBot.NET/Services/Triggers/Welcome.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.Threading.Tasks;
+using GlobalX.ChatBots.Core.Messages;
+using Microsoft.Extensions.Options;
+using PockyBot.NET.Configuration;
+using PockyBot.NET.Constants;
+using PockyBot.NET.Persistence.Repositories;
+
+namespace PockyBot.NET.Services.Triggers
+{
+    internal class Welcome : ITrigger
+    {
+        private readonly IConfigRepository _configRepository;
+        private readonly PockyBotSettings _pockyBotSettings;
+
+        public string Command => Commands.Welcome;
+        public bool DirectMessageAllowed => true;
+        public bool CanHaveArgs => false;
+        public string[] Permissions => Array.Empty<string>();
+
+        public Welcome(IOptions<PockyBotSettings> pockyBotOptions, IConfigRepository configRepository)
+        {
+            _configRepository = configRepository;
+            _pockyBotSettings = pockyBotOptions.Value;
+        }
+
+        public async Task<Message> Respond(Message message)
+        {
+            var newMessage = "## Hello world!\n" +
```

Comment:
if these kinds of strings were constants somewhere or in config files would it make them easier to find and edit if needed in the future?

Reply:
I'm not sure if it would be any easier - besides which the only place this is used is inside the Welcome trigger file - which is a very short and simple file.

I see your point, but given the amount of string messages in PockyBot as a whole, such a string constants file would likely become very large and confused.

Reply:
Wouldn't necessarily have to be a single file,  just a thought